composer.json

```diff
@@ -1,11 +1,11 @@
 {
     "config": {
         "platform": {
-            "php": "7.0.33"
+            "php": "7.2"
         }
     },
 	"require-dev": {
-		"phpunit/phpunit": "6 - 7",
+		"phpunit/phpunit": "6.5.14",
```

Comment:
Thank you. Do we need to make a new project to update PHPUnit? Theoretically, PHPUnit 8.x supports PHP 7.2, but it looks like v7.x is preferred. Did you happen to try v7.x or v8.x?

Reply:
I updated to PHPunit 7 and fix the errors. Can you take a look